src/test/java/com/palantir/docker/compose/service/ServiceClusterIntegrationTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.docker.compose.service;
+
+import com.palantir.docker.compose.DockerComposition;
+import com.palantir.docker.compose.connection.waiting.HealthChecks;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+public class ServiceClusterIntegrationTest {
+
+    private final DockerService postgresFromComposeFileSnippet = DockerService.fromDockerCompositionFile("src/test/resources/postgres-service.yaml")
+                                                                              .withHealthCheck("postgres", HealthChecks.toHaveAllPortsOpen());
```

Comment:
the `toHaveAllPortsOpen` is designed to be static imported with the `waitForService` method - we should either change `withHealthCheck` or `toHaveAllPortsOpen` - the combination of both is clunky.